crime-commons-schemas/build.gradle

```diff
@@ -0,0 +1,95 @@
+plugins {
+    id "com.javiersc.semver.project" version "0.5.0"
+    id "signing"
+    id "maven-publish"
+    id 'org.jsonschema2dataclass' version '6.0.0'
+}
+
+semver {
+    tagPrefix.set("crime-commons-schemas-")
+}
+
+java {
+    sourceCompatibility = '17'
+}
+
+configurations {
+    compileOnly {
+        extendsFrom annotationProcessor
+    }
+}
+
+def versions = [
+        jacksonVersion   : "2.17.0",
+        validationapiVersion: '2.0.1.Final'
+]
+
+dependencies {
+    implementation "com.fasterxml.jackson.core:jackson-databind:$versions.jacksonVersion"
+    implementation "javax.validation:validation-api:$versions.validationapiVersion"
```

Comment:
Is there a reason why we can't use `jakarta.validation:jakarta.validation-api` replacing the older `javax.validation`?

https://mvnrepository.com/artifact/jakarta.validation/jakarta.validation-api